clients/cli/src/clap_app.rs

```diff
@@ -890,6 +894,17 @@ pub fn app<'a>(
                         .takes_value(false)
                         .help("Enables group member configurations in the mint. The mint authority must initialize the member."),
                 )
+                .arg(
+                    Arg::with_name("ui_multiplier")
+                        .long("ui-multiplier")
+                        .value_name("UI_MULTIPLIER")
```

Comment:
nit: about being a little more verbose and say `ui-amount-multiplier`?
```suggestion
                    Arg::with_name("ui_amount_multiplier")
                        .long("ui-amount-multiplier")
                        .value_name("MULTIPLIER")
```